test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -525,19 +605,48 @@ bool VerifyUpdateToastProgressDataUsingValidTagAndEmptyGroup_Unpackaged()
 
     PostToastHelper(L"Tag", L"");
 
-    winrt::AppNotificationProgressData progressData = GetToastProgressData(L"SomeStatus", L"SomeTitle", 0.14, L"14%");
+    winrt::AppNotificationProgressData progressData = GetToastProgressData(L"SomeStatus", L"SomeTitle", 0.14, L"14%", true, 1);
 
-    auto progressResultOperation = toastNotificationManager.UpdateProgressDataAsync(progressData, L"Tag");
+    auto progressResultOperation = toastNotificationManager.UpdateAsync(progressData, L"Tag");
     return ProgressResultOperationHelper(progressResultOperation, winrt::AppNotificationProgressResult::Succeeded);
 }
 
+bool VerifyToastUpdateZeroSequenceFail_Unpackaged()
+{
+    try
+    {
+        Sleep(30000);
+        winrt::AppNotificationActivationInfo activationInfo(L"ToastNotificationApp", winrt::Uri{ LR"(C:\Windows\System32\WindowsSecurityIcon.png)" });
+
+        auto toastNotificationManager{ winrt::AppNotificationManager::Default() };
+
+        toastNotificationManager.Register(activationInfo);
+
+        auto scope_exit = wil::scope_exit(
+            [&] {
+                toastNotificationManager.Unregister();
+            });
+
+        PostToastHelper(L"Tag", L"");
+
+        winrt::AppNotificationProgressData progressData = GetToastProgressData(L"SomeStatus", L"SomeTitle", 0.14, L"14%", false, 1);
```

Comment:
sequenceNumber is set to 1 here? It won't throw right?

Reply:
No because the setsequence is set to false. It can be anything.